File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Lệnh rẽ nhánh 1/RENHANH1.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
    int n;
    cin >> n;
    if(n % 3 == 0)
        cout <<"{0} chia het cho 3";
    if(n % 3 == 2)
        cout <<"{0} chia 3 du 2";
    if(n % 3 == 1)
        cout <<"{0} chia 3 du 1";
    return 0;
}

File: Upcoder/Lập trình cơ bản/2.Lệnh rẽ nhánh/Lệnh rẽ nhánh 1/RENHANH1.cs
using System;
namespace Lenhrenhanh
{
    public class Program
    {
        static void Main(string[]args)
        {
            int n = int.Parse(Console.ReadLine());
            if(n % 3 == 0)
                Console.Write("{0} chia het cho 3", n);
            if(n % 3 == 2)
                Console.Write("{0} chia 3 du 2", n);
            if(n % 3 == 1)
                Console.Write("{0} chia 3 du 1", n);
        }
    }
}